@xen-orchestra/web-core/lib/components/ui/input/UiTextarea.vue

```diff
@@ -0,0 +1,103 @@
+<!-- v2 -->
+<template>
+  <div class="ui-textarea">
+    <VtsLabel v-if="slots.default" :accent="labelAccent" :required :icon :href><slot /></VtsLabel>
+    <textarea v-model="model" :class="accent" :disabled class="textarea" v-bind="$attrs" />
+    <VtsInfo v-if="slots.info" :accent><slot name="info" /></VtsInfo>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import VtsInfo from '@core/components/info/VtsInfo.vue'
+import VtsLabel from '@core/components/input/VtsLabel.vue'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { computed } from 'vue'
+
+defineOptions({
+  inheritAttrs: false,
+})
+
+const props = defineProps<{
+  accent: 'brand' | 'warning' | 'danger'
```

Comment:
Same, need to rebase and change `brand` to `info`
```suggestion
  accent: 'info' | 'warning' | 'danger'
```